### PR TITLE
BACK-1508: Integrate angle-staked-tokens

### PR DESCRIPTION
**Pull request by JackieJoo**

### CODE REVIEW

File: src/dex/angle-staked-stable/angle-staked-stable.ts [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+      return [
+        {
+          prices: amounts.map(amount =>
+            eventPool.getRateDeposit(amount, state),
+          ),
+          unit: eventPool.getRateDeposit(1n * BigInt(10 ** 18), state),
+          gasCost: AngleStakedGasCost,
+          exchange: exchange,
+          data: { exchange: `${stakeToken}` },
+          poolAddresses: [`${stakeToken}`],
```
Comment (Wertusser) [inline src/dex/angle-staked-stable/angle-staked-stable.ts:118]: typo: probably should be `${stakeToken}_${agToken}`?

Comment (JackieJoo) [reply src/dex/angle-staked-stable/angle-staked-stable.ts:118]: it's not used here, so I'll just remove it

Comment (JackieJoo) [reply src/dex/angle-staked-stable/angle-staked-stable.ts]: Fixed to have the right address for all cases

**Review by KanievskyiDanylo — APPROVED** [review]

File: src/dex/angle-staked-stable/angle-staked-stable.ts [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+import { AngleStakedStableEventPool } from './angle-staked-stable-pool';
+import StakedStableABI from '../../abi/angle/stagToken.json';
+
+const AngleStakedGasCost = 80000;
+
+export class AngleStakedStable
+  extends SimpleExchange
+  implements IDex<AngleStakedStableData>
+{
+  static readonly wstETHIface = new Interface(StakedStableABI);
```
Comment (nzrii-kysel) [inline src/dex/angle-staked-stable/angle-staked-stable.ts:30]: unused var?

**Review by nzrii-kysel — APPROVED** [review]

File: src/dex/angle-staked-stable/angle-staked-stable.ts (hunk 2 above)
Comment (JackieJoo) [reply src/dex/angle-staked-stable/angle-staked-stable.ts]: removed

**Review by aburkut — APPROVED** [review]

**Pull request merged**